packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -56,6 +66,29 @@ function RecipientPicker(props: RecipientProps) {
   const [isKeyboardVisible, setKeyboardVisible] = useState(false)
   const [isSendToAddressWarningVisible, setSendToAddressWarningVisible] = useState(false)
 
+  const kitCall = useAsync(async () => {
+    return await getContractKitAsync(false)
+  }, [])
+
+  const resolveKitCall = useAsync(async () => {
+    if (kitCall.result) {
+      const resolveGroup = new ResolveGroup([
+        new ResolveAddress(),
+        new ResolveNom({ kit: kitCall.result, contractAddress: getContractAddressFromEnv() }),
+      ])
+
+      return await resolveGroup.resolve(props.searchQuery)
+    }
+  }, [kitCall.result, props.searchQuery])
+
+  const getContractAddressFromEnv = () => {
```

Comment:
This looks like should be an environment variable instead?

I guess the drawback would be that we'd have to update this addresses ourselves instead of being automatic when the lib is updated.

If that's the case I think it looks nicer to create an object like this:
```
const NOM_ADDRESSES = {
  mainnet: ResolveNom.MainnetContractAddress,
  alfajores: ResolveNom.AlfajoresContractAddress,
}
```
and then just use `NOM_ADDRESSES[DEFAULT_TESTNET]` where we need it.

Reply:
I'm gonna change it to use the `NOM_ADDRESSES`, I don't like too much the idea of having a new env var.
Ideally, we shouldn't have to add extra info to our configs because of new integrations.
But if you feel that's the best way to do it, I'll change it.